lib/storage/index.js

```diff
@@ -131,6 +131,18 @@ function Storage(config) {
  * var storage = gcloud.storage();
  * var albums = storage.bucket('albums');
  *
+ * //-
+ * // Make a bucket's contents publicly readable.
```

Comment:
Can you elaborate here to say that new items added to the bucket will be made public by default?